samples/drivers/clock_control_litex/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  name: Sample for the clock control driver
+tests:
+  sample.driver.clock_control_litex:
+    build_only: true
```

Comment:
why build_only?

Reply:
if we connect one of those boards to sanitycheck, we should be able to build and run it, this otherwise will prevent that.